.rubocop_todo.yml

```diff
@@ -0,0 +1,65 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2024-07-31 10:08:15 UTC using RuboCop version 1.64.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 21
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyle, IndentationWidth.
+# SupportedStyles: with_first_argument, with_fixed_indentation
+Layout/ArgumentAlignment:
+  Exclude:
+    - 'lib/hammer_cli_foreman_discovery.rb'
+    - 'lib/hammer_cli_foreman_discovery/command_extensions/provision_with_puppet.rb'
+    - 'lib/hammer_cli_foreman_discovery/discovery.rb'
+    - 'lib/hammer_cli_foreman_discovery/discovery_rule.rb'
+    - 'test/unit/discovery_rules_test.rb'
+    - 'test/unit/discovery_test.rb'
+
+# Offense count: 2
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyle, IndentationWidth.
+# SupportedStyles: special_inside_parentheses, consistent, align_braces
+Layout/FirstHashElementIndentation:
+  Exclude:
+    - 'test/unit/discovery_resource_mock.rb'
+
+# Offense count: 15
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyle, EnforcedShorthandSyntax, UseHashRocketsWithSymbolValues, PreferHashRocketsForNonAlnumEndingSymbols.
+# SupportedStyles: ruby19, hash_rockets, no_mixed_keys, ruby19_no_mixed_keys
+# SupportedShorthandSyntax: always, never, either, consistent, either_consistent
+Style/HashSyntax:
+  Exclude:
+    - 'Rakefile'
+    - 'lib/hammer_cli_foreman_discovery.rb'
+    - 'lib/hammer_cli_foreman_discovery/discovery.rb'
+    - 'lib/hammer_cli_foreman_discovery/discovery_references.rb'
+    - 'lib/hammer_cli_foreman_discovery/discovery_rule.rb'
+
+# Offense count: 350
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyle, ConsistentQuotesInMultiline.
+# SupportedStyles: single_quotes, double_quotes
+Style/StringLiterals:
```

Comment:
Also updated some cops in todo file, which was causing major changes in the code, but what do you think of it?